kusto/ingest/streaming.go

```diff
@@ -50,48 +58,58 @@ func NewStreaming(client QueryClient, db, table string) (*Streaming, error) {
 // This method is thread-safe.
 func (i *Streaming) FromFile(ctx context.Context, fPath string, options ...FileOption) (*Result, error) {
 	props := i.newProp()
-	file, err := prepFileAndProps(fPath, &props, options, StreamingClient)
+	file, err, local := prepFileAndProps(fPath, &props, options, StreamingClient)
+
 	if err != nil {
 		return nil, err
 	}
 
-	return streamImpl(i.streamConn, ctx, file, props)
-}
+	if !local {
+		return streamImpl(i.streamConn, ctx, generateBlobUriPayloadReader(fPath), props, true)
+	}
 
-func prepFileAndProps(fPath string, props *properties.All, options []FileOption, client ClientScope) (*os.File, error) {
-	local, err := queued.IsLocalPath(fPath)
 	if err != nil {
 		return nil, err
 	}
 
+	return streamImpl(i.streamConn, ctx, file, props, false)
+}
+
+func prepFileAndProps(fPath string, props *properties.All, options []FileOption, client ClientScope) (*os.File, error, bool) {
```

Comment:
Mmmm, not sure how I feel about the third argument.
Maybe we'll have os.File just be nil if it's not local?

Reply:
This was what i had initially and Yocahi said its not readable

Reply:
A doc string would help clarify the meaning of each returned value